src/librustc_mir/interpret/snapshot.rs

```diff
@@ -285,12 +286,25 @@ impl<'a, Ctx> Snapshot<'a, Ctx> for &'a Allocation
     type Item = AllocationSnapshot<'a>;
 
     fn snapshot(&self, ctx: &'a Ctx) -> Self::Item {
-        let Allocation { bytes, relocations, undef_mask, align, mutability, extra: () } = self;
+        let Allocation {
+            relocations,
+            size,
+            align,
+            mutability,
+            extra: (),
+            ..
+        } = self;
+
+        let all_bytes = 0..self.len();
+        // FIXME: why is this access okay?
```

Comment:
It's OK because we only use it for detecting loops and we extract the undef mask and relocations separately

Reply:
Good to know, included that in the reasoning.